### DISCUSSION TITLE
Future plans for Integration / connection to trafic game planning tools

### DISCUSSION THREAD

**Discussion by AlexEhn** [tellurianinteractive/Tellurian.Trains.ModulesRegistryApp#43] [Q&A] [answered]
Hi, 

A question in advance to the Module zoom meeting tomorrow. Reply & discussion might be good to have in the zoom meeting. 

What are the future plans for possibility to use the data in the Module registry as input in game planning? 

Much of the data is of essence when planning a game and some existing trafic game planning applications (_at least one I know of_) have the same data as this module registry, but also connected to a game planning functionality.

**Comment by fjallemark** [comment, marked as answer]
When the planned features for the meeting management in the Module Registry is ready, there will be a first release of the planning tool just containing the first [waybill report](https://github.com/tellurianinteractive/Tellurian.Trains.TimetablePlanningApp/blob/master/Examples/Waybills%20(sv).pdf).  Future reports will be based on the local planning app. See examples [here](https://github.com/tellurianinteractive/Tellurian.Trains.TimetablePlanningApp/tree/master/Examples). There will also come tools for creating stretches, trains, schedules, duties and freight flow. Read more about the [background and concepts](https://github.com/tellurianinteractive/Tellurian.Trains.TimetablePlanningApp/wiki) for the Timetable Planning App.
Most work with the Timetable Planning App will be made during and after summer 2021.
